src/commands.ts

```diff
@@ -208,6 +164,30 @@ export async function openTriggerEvent() {
   }
 }
 
+export async function buildTriggerEventsList(events: string[], stripeClient: StripeClient):
+    Promise<vscode.QuickPickItem[]> {
+  const historicEvents = await stripeClient.getEvents();
```

Comment:
Unfortunately I forgot to mention that the Stripe CLI only supports triggering a subset of all the possible events. Does this return any event that is untriggerable? I fear that we may have to design this differently.

Reply:
Thanks for pointing that out, fixing!

Reply:
Also to answer the question, yes it can:
The list events can return events that aren't in our list: https://stripe.com/docs/api/events/types